Simplenote/src/main/java/com/automattic/simplenote/NoteEditorFragment.java

```diff
@@ -844,7 +844,9 @@ protected void showMarkdown() {
             new Runnable() {
                 @Override
                 public void run() {
-                    requireActivity().invalidateOptionsMenu();
+                    if(getActivity() != null) {
+                        getActivity().invalidateOptionsMenu();
+                    }
```

Comment:
When rotating the screen from portrait to landscape, then to portrait fast, this runnable will try to run after the fragment was detached.